qgis4/ansible/qgis4.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+USER=pblottiere
```

Comment:
Do you want to keep that?

Reply:
Good catch. What do you think about something like `USER=$1`?

Reply:
Sounds good.